care/facility/api/serializers/bed.py

```diff
@@ -108,6 +108,14 @@ def validate(self, attrs):
             start_date = attrs["start_date"]
             end_date = attrs.get("end_date", None)
             existing_qs = ConsultationBed.objects.filter(consultation=consultation, bed=bed)
+            latest_qs = ConsultationBed.objects.filter(consultation=consultation).latest("id")
```

Comment:
This logic will crash if there is no latest bed. you need to add a block that only runs if the latest bed exists.